FluentTerminal.RuntimeComponent/WebAllowedObjects/TerminalBridge.cs

```diff
@@ -9,19 +9,15 @@ namespace FluentTerminal.RuntimeComponent.WebAllowedObjects
 {
     internal static class EventDispatcher
     {
-        private static CoreDispatcher _dispatcher => CoreWindow.GetForCurrentThread()?.Dispatcher;
-
         public static async void Dispatch(Action action)
         {
-            // already in UI thread:
-            if (_dispatcher == null || _dispatcher.HasThreadAccess)
+            if (CoreWindow.GetForCurrentThread()?.Dispatcher is { } dispatcher)
```

Comment:
what is `{  }` ? Never seen this before

Reply:
It has the same meaning as `is CoreDispatcher`, but ReSharper prefers shorter syntax `is { }`. It's basically checking if it is `null` or not.